### ISSUE TITLE
cursor pointer is not visible on sign-in & need help buttons

### ISSUE THREAD

**Issue by nayyyhaa** [ahmadsohail404/Netflix-UI-Clone#51]
1) Go to the home screen
2) Click on the "Sign In" button
3) On the sign-in page, white hovering over the sign-in button and need help option.
4) Cursor pointer is missing which causes accessibility issues.

**PFA the SS**
**Actual**
1) No cursor on "Sign In" button.
![image](https://user-images.githubusercontent.com/46138150/135845768-14f5e8a1-d6fe-4ebb-ae07-7698775f0b87.png)

2) No cursor on "Need Help" option.
![image](https://user-images.githubusercontent.com/46138150/135845643-7b663fe9-1c17-4a75-8f21-296f52479e38.png)

**Expected**
Curson pointer should be there on "Sign-in" button and "Need Help" option.

**Comment by omeiirr** [comment]
Can I work on this issue?

**Comment by himanshu-malviya15** [comment]
@nayyyhaa I am interested in this issue please assign

**Comment by nayyyhaa** [comment]
@ahmadsohail404, I've created a PR #55 for this issue. Please review it :)

**Comment by ahmadsohail404** [comment]
@nayyyhaa attach an ss of what changes u made

**Comment by ahmadsohail404** [comment]
@nayyyhaa done :)
